make/configure.py

```diff
@@ -1917,10 +1922,15 @@ class Tools:
         doc.add( 'GCC.sysroot', '' )
         doc.add( 'GCC.minver', '' )
 
+    if build.match( '*-*-freebsd*' ):
```

Comment:
This should be relocated up a few lines with the other "HAS.*" definitions.  We like to keep related definitions grouped in the generated GNUmakefile.

Reply:
I found other `HAS.*` definitions in mingw section (if else group),
moved FreeBSD's definition following to them and changed if condition similar to mingw.
How about this?